### PR TITLE
Previous pull request minus CI

### LINKED ISSUE #6: Errors
I tried to use the class FinancialsDownloader() class and didnt work :(.

Code example below:

```
fd = gm.FinancialsDownloader()
test = fd._download('APPL', 'is')
```
and gives the error:

with urllib2.urlopen(url) as response:
NameError: name 'urllib2' is not defined ...

In order to fix the problem I changed the line 300 
FROM `with urllib2.urlopen(url) as response:` TO` with urllib.request.urlopen(url) as response:`

As soon as I overcomed the problem above I am getting the following error that is caused by line **302**    `json_data = json.loads(json_text)` :

json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

I am on a dead-end situation now :(

### PR DESCRIPTION
**Pull request by linwoodc3**
This PR fixes the bug and adds exception handling.  Unittest instructions included.

Regarding the CI, it wouldn't have added anything to the library download (did you put this on pypi?).  CI is just a way to ensure that libraries/code maintain their functionality when people add features to it.  Just makes sure the code still works when anyone adds new lines of code.  Good at pointing finger :-).  

Closes #6 and #7

### CODE REVIEW

File: morningstar/good_morning.py [hunk 1]
```diff
@@ -20,31 +20,32 @@
 """Module for downloading financial data from financials.morningstar.com.
 """
 
-from __future__ import with_statement
 from __future__ import absolute_import
```
Comment (petercerno) [inline morningstar/good_morning.py]: Where is this used?

File: README.md [hunk 2]
```diff
@@ -13,16 +13,16 @@ Motivation
 
 Good Morning is intended to be used as an extension to [QSToolKit (QSTK)](http://wiki.quantsoftware.org/index.php?title=QuantSoftware_ToolKit) library. By using [QSTK](http://wiki.quantsoftware.org/index.php?title=QuantSoftware_ToolKit) you can easily download historical stock market data from [Yahoo Finance](http://finance.yahoo.com/). You can also download fundamental financial data from [Compustat](https://www.capitaliq.com/home/what-we-offer/information-you-need/financials-valuation/compustat-financials.aspx). However, most individuals and institutions do not have access to [Compustat](https://www.capitaliq.com/home/what-we-offer/information-you-need/financials-valuation/compustat-financials.aspx). Good Morning attempts to mitigate this limitation by providing a very simple Python interface for downloading fundamental financial data from [financials.morningstar.com](http://financials.morningstar.com/).
 
-Example
-=======
+Example of Downloading Key Ratios from MorningStar
+==================================================
 
-    import good_morning as gm
+    import morningstar as gm
```
Comment (petercerno) [inline README.md]: Is this correct? I do not see any file named morningstar

File: tests/test_download.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+from unittest import TestCase
+
+from morningstar import good_morning as gm
```
Comment (petercerno) [inline tests/test_download.py]: Is this import working? I don't see any file named morningstar

File: README.md (hunk 2 above)
Comment (linwoodc3) [inline README.md]: Yes, @petercerno, this code works.  This isn't a file, but it's  folder to hold a module.  The `__init__.py` file in the folder is what makes the code inside the folder importable.  

The CI test I had made sure it worked.  And, even without the CI test, if you install this code, and run the example in the updated README, it works. 

To give some info on the CI code, this is exactly what CI code is meant to support.  I runs automatic tests when someone adds/pushes code to github.

**Comment by linwoodc3** [comment]
Thanks for reviewing !

Those commits are gone; they are just in the record.  If you click the "View changes" button, that's all that changes.

Nothing else.  Git just keeps a history of everything.  This is ALL that changed: https://github.com/petercerno/good-morning/pull/9/files/a04d633cc63ae2ff7d9b9265e11c4f9d5a704c99

Red is code that is deleted.  Green is code that is added.

All of the CI code is gone; it's just old history.  You can see that I only added the bugfix and exception handling in the View changes page.

File: tests/test_download.py (hunk 3 above)
Comment (linwoodc3) [inline tests/test_download.py]: Yes, it works.  If you install the library from my pull request, you should be able to run through the example.  I added this folder in the last pull request a month ago.

File: morningstar/good_morning.py (hunk 1 above)
Comment (linwoodc3) [inline morningstar/good_morning.py]: I'll delete this; but I was going to make this code work with Python 2 eventually (next plan) for backwards compatibility.  

http://python-future.org/quickstart.html

**Comment by linwoodc3** [comment]
I'll just redo it.  Don't want to confuse.

**Pull request closed without merging**